### DISCUSSION TITLE
I don't receive my email notifications anymore

### DISCUSSION THREAD

**Discussion by lindenb** [community/community#22309] [New to GitHub]
<p>Hi all,</p>
<p>I used to received my email-notifcations from github, but I think I haven’t  received any notification from github via my email about the issues of my project  since november : e.g: <a href="https://github.com/lindenb/jvarkit/issues/88" rel="noopener nofollow ugc">https://github.com/lindenb/jvarkit/issues/88</a> or about the projects I’m watching e.g: <a href="https://github.com/samtools/htsjdk/pulls" rel="noopener nofollow ugc">https://github.com/samtools/htsjdk/pulls</a></p>
<p>* I’ve checked my spam folder and my trash</p>
<p>* there is no ‘hard’ filter in my inbox for github</p>
<p>* in settings : "Receive all emails, except those I unsubscribe from. " is checked</p>
<p>* I’ve checked my email (+ primary + notifications)</p>
<p>(you can test it : <a href="https://github.com/lindenb/jvarkit/issues/91" rel="noopener nofollow ugc">https://github.com/lindenb/jvarkit/issues/91</a> )</p>
<p>Thanks in advance for the help.</p>

**Comment by manisha-temkar-sp** [comment]
<p>I am also facing same issue. Please suggest ASAP.</p>

**Comment by nadiajoyce** [comment, marked as answer]
<p><a class="mention" href="/lindenb">@lindenb</a> and <a class="mention" href="/manisha-temkar-sp">@manisha-temkar-sp</a> If you were successfully receiving email notifications previously and suddenly are not, I would recommend reaching out to private support at <a href="http://git.io/c" class="inline-onebox-loading" rel="nofollow noopener">git.io/c</a>. GitHub staff may need to take a look at your specific account details to help find a resolution.</p>

**Comment by lindenb** [comment]
<p>thanks <a class="mention" href="/nadiajoyce">@nadiajoyce</a> I’ve just send a message.</p>
<p>BTW: I’ve received a email-notification for *your* message :slight_smile:</p>

**Comment by nadiajoyce** [comment]
<p><a class="mention" href="/lindenb">@lindenb</a> Great! And you probably received an email notification from my response here, because the email notification settings for the Community Forum are separate from those on <a href="http://github.com" rel="nofollow noopener">github.com</a>. I think someone in private support will be able to help you with your <a href="http://github.com" rel="nofollow noopener">github.com</a> email notification settings after taking a closer look at your account. Cheers!</p>

**Comment by lindenb** [comment]
<p><span class="mention">@nadiajoycethis</span> has been just fixed by your co-worker. Strangely my account was flagged as ‘spam’ whereas I’m a ‘kind/normal’ user :slight_smile:</p>
<p>Thank you again for the help.</p>

**Comment by KimGuevarra** [comment]
<p>    </p>

**Comment by AlexAndBear** [comment]
Facing the same issue

**Comment by GitAlexToTheGreek** [comment]
Same thing; notifications are enabled in Settings, but not receiving alerts when a reply is made to my subscribed post in the "Issues" section.

Nothing found in the email's Spam section either. 

Can anyone assist?

**Comment by eitch** [comment]
This issue is plaguing me too since a couple of weeks.
